packages/mgt-components/src/styles/shared-sass-variables.scss

```diff
@@ -47,6 +47,7 @@ $greyHover: #f7f7f7; /* Person Card Hover */
 $greyBackground: #fbfbfb; /* Person Card Background */
 $commblue_org: #0360b9; /* Person Card Org border */
 $gray_separator_org: #d6d6d6; /* Person Card Org target borders */
+$showMoreButtonHover: #efefef; /* File List Hover */
```

Comment:
@Mnickii this is a nitbit on the variable naming in scss/css, let us use  snake_case over camelCase.